.github/auto-merge.yml

```diff
@@ -90,14 +82,6 @@
     dependency_name: Particular.PlatformSample.ServicePulse
     update_type: all
 
-- match:
-    dependency_name: PublicApiGenerator
-    update_type: all
-
```

Comment:
https://github.com/Particular/ServiceControl.Monitoring.Data/blob/master/src/ServiceControl.Monitoring.Data.Tests/ServiceControl.Monitoring.Data.Tests.csproj#L16

Reply:
PublicAPIGenerator should stay. It is widely used in our codebase, this one project does not change a thing. Updating the dependency to version 10 and greater causes the project to not compile => that PR will not be automatically merged

Reply:
Actually, it's restricting the upper range anyway, so this doesn't have to be removed. I'll revert.